ligo/src/registryDAO/types.mligo

```diff
@@ -20,6 +20,7 @@ type proposal_type =
   | Normal_proposal of registry_diff
   | Update_receivers_proposal of update_receiver_param
   | Configuration_proposal
+  | Transfer_proposal of transfer_type list
 
 type lookup_registry_param =
```

Comment:
Would you please add a "For reference only" `proposal_metadata` here as well as we have in `treasuryDAO/types.mligo` as it's getting difficult to reason about it otherwise?